db/change_cache.go

```diff
@@ -468,13 +469,24 @@ func (c *changeCache) DocChangedSynchronous(event sgbucket.FeedEvent) {
 	}
 
 	if syncData.Sequence <= c.getInitialSequence() {
-		return // Tap is sending us an old value from before I started up; ignore it
+		return // DCP is sending us an old value from before I started up; ignore it
 	}
 
-	// Record a histogram of the Tap feed's lag:
-	tapLag := time.Since(syncData.TimeSaved) - time.Since(entryTime)
-	lagMs := int(tapLag/(100*time.Millisecond)) * 100
-	changeCacheExpvars.Add(fmt.Sprintf("lag-tap-%04dms", lagMs), 1)
+	// Measure feed latency from timeSaved or the time we started working the feed, whichever is later
+	var feedLatency time.Duration
+	if !syncData.TimeSaved.IsZero() {
+		c.context.DbStats.StatsCblReplicationPull().Add(base.StatKeyDcpReceivedCount, 1)
```

Comment:
The comment above suggested you wanted to increment this count, even when `syncData.TimeSaved.IsZero()`. Do we need to move this up out of the if ?